buildpack.toml

```diff
@@ -11,8 +11,9 @@ keywords = ["procfile", "processes", "heroku"]
 [[buildpack.licenses]]
 type = "BSD-3-Clause"
 
-[[stacks]]
-id = "*"
+[[targets]]
+os = "linux"
+arch = "amd64"
```

Comment:
Should we omit the `arch`, given the buildpack only need `bash` (so linux) but is otherwise architecture independent?

Reply:
Since the buildpack is compiled to `x86_64-unknown-musl`, it is architecture dependent?

Reply:
As long as this buildpack is compiled, I think we'll always need to specify `arch`. It's unlikely that we'd ever compile this buildpack to `x86` (32-bit), `ppc64el`, or`S390X` (all are supported by ubuntu).

Reply:
Good point!